sleep_rate: 0.047
